x-pack/plugins/security_solution/public/management/pages/policy/view/ingest_manager_integration/endpoint_policy_create_extension.tsx

```diff
@@ -5,45 +5,461 @@
+              </RadioOptionsDetails>
+            </IntegrationOptionsContainer>
+          </>
+        )}
+
+        {dropdownValue === dropDownOptions[1].value && (
+          <>
+            <IntegrationOptionsContainer>
+              <CloudOptionDataIngestionMessage>
+                <FormattedMessage
+                  id="xpack.fleet.createPackagePolicy.stepConfigure.interactiveSessionSuggestionTranslation"
+                  defaultMessage="To save on data ingestion volume select interactive session only "
+                />
+              </CloudOptionDataIngestionMessage>
+
+              <EuiRadio
+                id="radioInteractiveOption"
+                label="Interactive only"
+                checked={radioInteractiveSelected}
+                onChange={onChangeRadio}
+              />
+              <RadioOptionsDetails>
+                <FormattedMessage
+                  id="xpack.fleet.createPackagePolicy.stepConfigure.packagePolicTypeInteractiveOnlyInfo"
+                  defaultMessage="Monitors and collects session data from interactive sessions only. "
+                />
+              </RadioOptionsDetails>
+
+              <EuiRadio
+                id="radioComprehensiveOption"
+                label="All events"
+                checked={radioComprehensiveSelected}
+                onChange={onChangeRadio}
+              />
+              <RadioOptionsDetails>
+                <FormattedMessage
+                  id="xpack.fleet.createPackagePolicy.stepConfigure.packagePolicTypeComprehensiveInfo"
+                  defaultMessage="Monitors and collects session data from all process executions. "
+                />
+              </RadioOptionsDetails>
+
+              <BoldSubtitle>
+                <FormattedMessage
+                  id="xpack.fleet.createPackagePolicy.stepConfigure.protectionModeTranslation"
+                  defaultMessage="Protection mode"
+                />
+              </BoldSubtitle>
+              <SubduedText>
+                <FormattedMessage
+                  id="xpack.fleet.createPackagePolicy.stepConfigure.protectionModeDetailsTranslation"
+                  defaultMessage="In addition to detections, Elastic security can prevent threats before they happen.
+              You can disable detections anytime in the agent policy configurations settings."
+                />
+              </SubduedText>
+              <CloudRadioProtectionsModeContainer>
+                <EuiFlexGroup>
+                  <EuiFlexItem>
+                    <EuiCheckbox
+                      id="CheckBoxIdMalware"
+                      label="Prevent malware"
+                      checked={checkboxMalwareChecked}
+                      onChange={onChangeMalwareCheckbox}
+                    />
+                  </EuiFlexItem>
+                  <EuiFlexItem>
+                    <EuiCheckbox
+                      id="CheckBoxIdRansomware"
+                      label="Prevent ransomware"
```

Comment:
Remove Ransomware, and add `Malicious Behavior` instead (Ransomware protection is currently not supported on the Linux cloud environment)